recipe/recipe.yaml

```
@@ -0,0 +1,49 @@
+schema_version: 1
+
+context:
+  name: hepunits
+  version: 2.3.5
+  python_min: 3.8
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/hepunits-${{ version }}.tar.gz
+  sha256: 9434cd2d6a722c949e9e9e22bd0b59587f69025bd37356e5c70635f1b37036d8
+
+build:
+  number: 2
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python 3.8.*
```

```suggestion
    - python ${{ python_min }}.*
```

Currently there's a bug for all of these that the `python_min` doesn't get properly replaced everywhere, so you need to manually replace the `${{ python_min}}.*` in places like this.

c.f. https://github.com/conda-incubator/conda-recipe-manager/issues/307